examples/test_integration.py

```diff
@@ -1,9 +1,22 @@
+""" Acquiring the Error Associated for Experimental Data
+========================================================
+
+Takes data from a standard echo experiment with multiple scans and compares the 
+traditional method of acquiring the error, the propagated error along the inactive
+coherence transfer (CT) pathways in the integrated signal slice, and the error associated
+with the integrated signal in the active coherence pathway.
```

Comment:
these are all methods of estimating the error of the integral.  That's not clear here.

Also, you don't describe what the "traditional method" does.

Also, "propagated error" is not a sufficient description.  You propagate the error of XXX to estimate the error associated with YYY.

Reply:
okay I'm hoping the new docstring is a little clearer....

Reply:
yes, I edited slightly, but good